steward/src/commands/cork_proposal/aave_v2_stablecoin/set_fees_distributor.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{application::APP, cellars, commands::cork_proposal::print_proposal, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use deep_space::Address;
+use steward_proto::steward::{
+    aave_v2_stablecoin_governance::{Function, SetFeesDistributor},
+    governance_call::Call,
+    AaveV2StablecoinGovernance, GovernanceCall,
+};
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nSet fees distributor Cosmos address of target cellar when chain reaches specified height"
+)]
+pub struct FeesDistributorCmd {
+    #[clap(short, long)]
+    /// Fee distributor's address
+    new_fees_distributor: Address,
```

Comment:
Nit: it's a tad confusing that all the prop commands import `Address` but in some it means a Cosmos address and in some it means an Ethereum address. It might be clearer to use a named import, e.g. `as CosmosAddress`.